src/components/mote/components/innkalling/NyttDialogMote.tsx

```diff
@@ -0,0 +1,152 @@
+import { ArbeiderKvinneImage } from "../../../../../img/ImageComponents";
+import { Flatknapp, Knapp } from "nav-frontend-knapper";
+import ModalWrapper from "nav-frontend-modal";
+import React, { ReactElement, useState } from "react";
+import styled from "styled-components";
+import { useHistory } from "react-router";
+import { useParams } from "react-router-dom";
+import Veileder from "nav-frontend-veileder";
+import { FlexColumn, FlexRow, ModalContentContainer } from "../../../Layout";
+import { erLokalEllerPreprod } from "../../../../utils/miljoUtil";
+
+const ModalText = styled.div`
+  max-width: 40ch;
+`;
+
+export const NyttDialogMote = (): ReactElement => {
+  const [behandlerModalIsOpen, setBehandlerModalIsOpen] = useState(false);
+  const [nyLosningModalIsOpen, setNyLosningModalIsOpen] = useState(false);
+  const history = useHistory();
+  const { fnr } = useParams();
+
+  const texts = {
+    nyttMote: "Nytt dialogmøte",
+    behandlerVaereMed: "Skal behandleren være med i dialogmøtet?",
+    nyLoesningInnkalling: "Ny løsning for innkalling til Dialogmøte",
+    arbeiderBilde: "Bilde av arbeider",
+    nei: "Nei",
+    ja: "Ja",
+    avbryt: "Avbryt",
+    modalDM2Info:
+      "Vi utvikler en ny løsning der du kan kalle inn til dialogmøte og skrive referat direkte i Modia.",
+    modalOnskerDuProve: "Ønsker du å prøve ut den nye løsningen",
+  };
+
+  const dm1 = (
```

Comment:
Om dette er dialogmotemoteplanleggerer, foreslår jeg å kalle det for å gjøre det lettere å forstå hva det er

Tilsvarende for dialogmoteinnkalling under(dm2).

Reply:
Skjønner! Det er link til dialogmøteplanleggeren. Kan rename